### PR TITLE
add fuse online alerts

### PR DESCRIPTION
**Pull request by StevenTobin**
## Additional Information
Add monitoring alerts for Fuse Online

## Verification Steps
1. Install from this branch
2. Ensure there are three alerts added to the middleware-monitoring alertmanager
> FuseOnlineSyndesisServerInstanceDown 
> FuseOnlineSyndesisUIInstanceDown
3. Scale down the syndesis-db, syndesis-ui and syndesis-server pods
4. Verify that the alerts trigger

## Is an upgrade task required and are there additional steps needed to test this?

- [x] Tested Installation
- [x] Tested Uninstallation
- [ ] Tested or Created follow on for Upgrade

### CODE REVIEW

**Comment by damienomurchu** [comment]
Just installing to a cluster now to verify

File: roles/middleware_monitoring_config/templates/kube_state_metrics_fuse_online_alerts.yml.j2 [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+spec:
+  groups:
+    - name: fuseOnline.rules
+      rules:
+        - alert: FuseOnlineSyndesisServerInstanceDown
+          annotations:
+            message: >-
+              Fuse Online Syndesis Server instance {{ "{{" }}$labels.pod{{ "}}" }} in namespace {{ "{{" }}$labels.namespace{{ "}}" }} is down.
+          expr: >
+            absent(kube_pod_status_ready{namespace="fuse", condition="true", pod=~"syndesis-server-.*"})
```
Comment (david-martin) [inline roles/middleware_monitoring_config/templates/kube_state_metrics_fuse_online_alerts.yml.j2]: namespace should be templated to allow for the ns_prefix

**Comment by damienomurchu** [comment]
Verified below on a PDS cluster:
- [] Installation
- [] Uninstall

Can confirm that alerts work as intended.

**Comment by StevenTobin** [comment]
Based on comments in upstream issue, I have removed the db instance down check as the imported check is in fact sufficient.

**Pull request merged**